docs/phylum_project_set-thresholds.md

```diff
@@ -5,12 +5,19 @@ hidden: false
 ---
 Interactively set risk domain thresholds for a project
 ```sh
-phylum project set-thresholds <name>
+phylum project set-thresholds <name> [OPTIONS]
```

Comment:
And another instance where I'd move the options to the start. This also applies to the examples and is in line with how clap generates the help and how other unix tools behave.

Reply:
fixed